src/test/java/seedu/address/testutil/PersonUtil.java

```diff
@@ -24,7 +24,7 @@ public class PersonUtil {
      * Returns an add command string for adding the {@code person}.
      */
     public static String getAddCommand(Person person) {
```

Comment:
you can change the method signature to `getAddPersonCommand` as well.
```suggestion
    public static String getAddPersonCommand(Person person) {
```

Reply:
thanks for catching!